extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -24,6 +24,10 @@ amp-sticky-ad {
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
   margin-bottom: 0 !important;
+  margin-left: 0 !important;
```

Comment:
This is for `amp-sticky-ad` which will always has `width:100%` with `margin-left` for example close button could be hidden.